cmd/config_delete.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/core/loadpoint"
+	"github.com/evcc-io/evcc/util/config"
+	"github.com/evcc-io/evcc/util/templates"
+	"github.com/spf13/cobra"
+)
+
+// configDeleteCmd represents the configure command
+var configDeleteCmd = &cobra.Command{
+	Use:   "delete <id>",
+	Short: "Delete device",
+	Run:   runConfigDelete,
+	Args:  cobra.ExactArgs(1),
+}
+
+func init() {
+	configCmd.AddCommand(configDeleteCmd)
+}
+
+func runConfigDelete(cmd *cobra.Command, args []string) {
+	// load config
+	if err := loadConfigFile(&conf, !cmd.Flag(flagIgnoreDatabase).Changed); err != nil {
+		log.FATAL.Fatal(err)
+	}
+
+	// setup environment
+	if err := configureEnvironment(cmd, &conf); err != nil {
+		log.FATAL.Fatal(err)
+	}
+
+	id, err := strconv.Atoi(args[0])
+	if err != nil {
+		log.FATAL.Fatal(err)
+	}
+
+	c, err := config.ConfigByID(id)
+	if err != nil {
+		log.FATAL.Fatal(err)
+	}
+
+	fmt.Println(strconv.Itoa(c.ID)+":", c.Type, c.Value)
```

Comment:
Lass uns hier die gleichen Identifier verwenden wie bei den anderen Befehlen (`evcc meter db:22`)

```suggestion
	fmt.Println("db:"+strconv.Itoa(c.ID)+":", c.Type, c.Value)
```